dist/bin/no-internal-report.js

```diff
@@ -1,64 +1,39 @@
 #! /usr/bin/env node
-
 /*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 "use strict"
 const path = require('path');
-const fs = require('fs');
-
-function getNodeModulesDir(startPath) {
-  let currPath = startPath;
-  let baseName = path.basename(currPath);
-  while (baseName !== "node_modules") {
-    currPath = path.resolve(currPath, "..")
-    if (currPath === "/")
-      return undefined;
-    baseName = path.basename(currPath);
-  }
-  return currPath;
-}
-
-const nodeModules = getNodeModulesDir(path.dirname(process.argv[1]));
-if (!nodeModules)
-  throw ("Could not find node_modules directory");
+const assert = require('assert');
+const eslintBinPath = path.join(require.resolve("eslint"), "../../bin/eslint.js");
 
-const distDir = path.join(nodeModules, "@itwin/eslint-plugin/dist")
-if (!fs.existsSync(distDir))
-  throw ("Could not find required dir: " + distDir);
-
-const configDir = path.join(nodeModules, "@itwin/eslint-plugin/dist/bin/eslint.config.js")
-if (!fs.existsSync(distDir))
-  throw ("Could not find required dir: " + configDir);
-
-let tags;
-if (process.argv.length > 3) {
-  tags = process.argv[2].toString();
+const argCount = process.argv.length;
+try {
+  assert(argCount >= 3 && argCount <= 4);
+} catch(error) {
+  console.error("Incorrect number of arguments");
+  console.error("Usage: no-internal-report [tags: internal,alpha,beta,public] [FILE_GLOBS]");
+  console.error("The 'tag' argument may be one or more comma-separated api tag names")
+  process.exit(1);
 }
 
+const tags = argCount == 3 ? "" : process.argv[2].split(",");
```

Comment:
Don't use nonstrict equal operator (==), and don't default to an empty string, prefer undefined

Reply:
no longer used due to above changes